### ISSUE TITLE
/register route not woking 

### ISSUE THREAD

**Issue by sudhir500** [RebbajAlan/rebbajalan.github.io#1]
hi,

I follow all steps are u described. i am able to login successfully. 
But i am getting problem in rest of the route. 
e.g when ii trying to access "/register" my server is taking too much time and finally i am getting 
<pre>
"The connection was reset" 
The connection to the server was reset while the page was loading.
</pre>
![image](https://cloud.githubusercontent.com/assets/4952560/16041964/f4aacd8c-3255-11e6-90e3-b9cc324b9c6c.png)

may u  help  me how to resolve this issue?

**Comment by RebbajAlan** [comment]
Hello,

I haven't come across this issue myself unfortunately.

If you're adding registration/login to an existing project, I recommend re-testing in a new blank Symfony project so that you can rule-out any conflicts with your existing code.

If it happens in a fresh new blank Symfony project as well, check that you're using the latest version of PHP (5.6 / 7.0), and that you're using the latest version of Symfony.

If it's still happening after that, and the PHP error logs on your server don't offer any clues, I recommend posting to Stackoverflow where they may be able to help you track down the cause.

**Comment by sudhir500** [comment]
i am using PHP 5.5.12. and symfony 2.8.7.
also i tested this code in symfony 3.1 but output was same as before.

**Comment by MrBeerBrewer** [comment]
I agree with RebbajAlan. It's working wonderfully. There is one small glitch. That has to do with 'homepage' redirect that can be solved quickly.

**Comment by sudhir500** [comment]
Hi +MrBeerBrewer 
Thanks for guiding me.  its working now. :-)    :+1:
